src/HealthGPS/healthgps.cpp

```diff
@@ -331,13 +331,13 @@ std::map<std::string, core::UnivariateSummary> HealthGPS::create_input_data_summ
     auto visitor = UnivariateVisitor();
     auto summary = std::map<std::string, core::UnivariateSummary>();
     auto &input_data = definition_.inputs().data();
+
     for (const auto &entry : context_.mapping()) {
-        try {
-            input_data.column(entry.name()).accept(visitor);
+        // HACK: Ignore missing columns.
+        auto &[success, column] = input_data.column_if_exists(entry.name());
+        if (success) {
+            column.accept(visitor);
```

Comment:
If you follow my suggestion of using `std::optional`, this could become:

```suggestion
        // HACK: Ignore missing columns.
        if (auto column = input_data.column_if_exists(entry.name())) {
            column->accept(visitor);
```